### PR TITLE
feat: add in-memory sequencer

### PR DESCRIPTION
**Pull request by stulzq**
**What this PR does**:
Add in-memory sequencer
**Which issue(s) this PR fixes**:

None
**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: components/go.mod [hunk 1]
```diff
@@ -31,6 +31,7 @@ require (
 	go.etcd.io/etcd/client/v3 v3.5.0
 	go.etcd.io/etcd/server/v3 v3.5.0
 	go.mongodb.org/mongo-driver v1.8.0
+	go.uber.org/atomic v1.7.0
```
Comment (seeflood) [inline components/go.mod:34]: @wenxuwan 我go水平有限，王老师看下这里引入 go.uber.org/atomic 是否合适

**Review by seeflood — APPROVED** [review]
lgtm

**Review by zhenjunMa — APPROVED** [review]
Sorry for the late reply  and thanks for your contribution!

**Comment by seeflood** [comment]
This component can make debug easier.Thanks !

**Pull request merged**